server/views.py

```diff
@@ -182,6 +183,95 @@ def card(request, tool_id, card_id):
 
   return makeResponse(request, result)
 
+@check_secret
+@check_ip
+@require_GET
+def getinfo(request, tool_id, item_requested):
```

Comment:
How does it know what items to request? I feel like an inventory method that just dumps the current tool's contents is the way forward here

Reply:
Now just dumps the contents of a tool